advanced/advanced_numpy/examples/stride-diagonals-answer.py

```diff
@@ -5,6 +5,7 @@
 Solution showing how to use as_strided to stride in diagonal.
 
 """
+from __future__ import print_function
```

Comment:
Actually, you don't need this line at all: the code in this example is compatible Python 2 and Python 3. So let's not even add this line.